functions/daily_schedule.pp

```diff
@@ -0,0 +1,9 @@
+# @param seed A seed to pass to `fqdn_rand`.  A good choice is the repo name that this schedule will be for.
+# @return Returns a 'pseudo-random' daily iso8601 schedule.
+# @example Calling the function
+#    pulp::daily_schedule('centos-7')
+function pulp::daily_schedule(String $seed) >> String {
```

Comment:
I've actually been using this function for a long time in my profile module.  Thought it was time to move it somewhere more appropriate. :)